superset-frontend/src/components/ImportModal/index.tsx

```diff
@@ -20,7 +20,7 @@ import React, { FunctionComponent, useEffect, useRef, useState } from 'react';
 import { styled, t } from '@superset-ui/core';
 
 import Icon from 'src//components/Icon';
-import StyledModal from 'src/common/components/Modal';
+import StyledModal from 'src/components/Modal';
```

Comment:
Not in love with this name (it's inconsistent with the other implementations) which is why I'm not really into default exports, personally. Not a concern for this PR, just whining :)

Reply:
Don't worry! My next step after organizing the folder will be to create an `index` file and export all components using named exports 😉.